drivers/staging/vc04_services/interface/vchiq_arm/vchiq_core.c

```diff
@@ -2097,7 +2097,6 @@ slot_handler_func(void *v)
 					vchiq_log_error(vchiq_core_log_level,
 						"Failed to send RESUME "
 						"message");
```

Comment:
Unless it causes problems during the upstreaming I'd be tempted to leave it, but we wouldn't miss it if it went.